optaplanner-website-root/content/learn/useCases/employeeRostering.adoc

```diff
@@ -1,29 +0,0 @@
-= Employee rostering
-:jbake-type: useCaseBase
-:jbake-description: OptaPlanner is an Open Source Java™ engine to optimize employee timetabling and shift rostering.
-:jbake-priority: 1.0
-:jbake-related_tag: employee rostering
-:jbake-featured_youtube_id: 3CvadujUN1k
-:jbake-featured_quickstart_github_url: https://github.com/kiegroup/optaweb-employee-rostering#readme
-
-image:employeeRosteringValueProposition.png[Employee rostering value proposition]
```

Comment:
Needs to stay. Still makes sense in use case explenation way. not optaweb specific